examples/kitchen-sink.yaml

```diff
@@ -289,8 +289,45 @@ tracer_provider:
           ratio: 0.01
       local_parent_sampled:
         composite/development:
-          probability:
-            ratio: 0.001
+          # Configure sampler to be rule_based.
+          rule_based:
+            # The rules for the sampler, matched in order. Each rule can have multiple match conditions - the sampler will be applied if all match. 
+            # If no conditions are specified, the rule matches all spans that reach it. If no rules match, the span is not sampled.
+            rules:
+              - attribute_values:
+                  key: http.route
+                  values:
+                    - /healthz
+                    - /livez
+                # Configure sampler when matched.
+                sampler:
+                  # Configure sampler to be always_off if matched.
+                  always_off: # Configure a sampling rule matching http.path attribute patterns.
+              - attribute_patterns:
+                  key: http.path
+                  patterns:
```

Comment:
I wonder if "patterns:" can be eliminated from the schema. The context is already about patterns.